components/brave_federated/features.cc

```diff
@@ -12,6 +12,7 @@ namespace features {
 
 namespace {
 
+// TODO(Moritz Haller): make ticket: rename to "BraveFederated"
```

Comment:
created the ticket https://github.com/brave/brave-browser/issues/21104

Reply:
@moritzhaller is there a reason we would not make this change before release?

Reply:
no real reason - changing it in here and added ticket as auto-closing keyword